.github/workflows/on-pr-dep-change.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
     if: ${{ always() }}
     needs:
       - dependency-tests
+      - create-project
```

Comment:
Since I removed pdoc, this workflow got triggered and I noticed this mistake in my recent "create a project for every test run" PR.